sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -91,32 +95,88 @@ credential.update("{new_api_key}");
 
 ## Key concepts
 ### Client
-The Form Recognizer client library provides a [FormRecognizerClient][form_recognizer_sync_client] and 
-[FormRecognizerAsyncClient][form_recognizer_async_client]. It provides both synchronous and
-asynchronous operations to access a specific use of Form Recognizer, such as training custom models,
-extracting layout and receipt information from documents.
+The Form Recognizer client library provides a [FormRecognizerClient][form_recognizer_sync_client], 
+[FormRecognizerAsyncClient][form_recognizer_async_client] and [FormTrainingClient][form_training_sync_client],
+[FormTrainingAsyncClient][form_training_async_client]. It also supports model management operations to
+get custom model info, delete model, list models and get account details.
+
+### FormRecognizerClient
+A `FormRecognizerClient` and `FormRecognizerAsynClient` provides both synchronous and asynchronous operations to access
```

Comment:
Just as a heads up, talking to @annelo-msft, the ordering from most used to least is something like:
recognize custom forms, recognize content, recognize receipt, train unlabeled, train labeled, manage custom models.
Would be good to have them in this order throughout the readme